static/app/views/profiling/content.tsx

```diff
@@ -137,6 +139,28 @@ function ProfilingContent({location}: ProfilingContentProps) {
     );
   }, [selection.projects, projects]);
 
+  const withoutProfilingSupport = useMemo((): boolean => {
+    const projectsWithProfilingSupport = new Set(
+      projects
+        .filter(project => !project.platform || profiling.includes(project.platform))
+        .map(project => project.id)
+    );
```

Comment:
i would maybe put all this inside UnsupportedAlert? Doesn't seem useful at the top level, UnsupportedAlert can choose when to display

Reply:
or maybe not UnsupportedAlert but another small component